ocis-pkg/CHANGELOG.md

```diff
@@ -0,0 +1,318 @@
+# Changelog for [unreleased] (UNRELEASED)
+
+The following sections list the changes in ocis-pkg unreleased.
+
+[unreleased]: https://github.com/owncloud/ocis-pkg/compare/v2.4.0...master
+
+## Summary
+
+* Change - Unwrap roleIDs from access-token into metadata context: [#59](https://github.com/owncloud/ocis-pkg/pull/59)
+* Change - Provide cache for roles: [#59](https://github.com/owncloud/ocis-pkg/pull/59)
+* Change - Roles manager: [#60](https://github.com/owncloud/ocis-pkg/pull/60)
+
+## Details
+
+* Change - Unwrap roleIDs from access-token into metadata context: [#59](https://github.com/owncloud/ocis-pkg/pull/59)
+
+   We pass the RoleIDs from the access-token into the metadata context.
+
+   https://github.com/owncloud/ocis-pkg/pull/59
+
+
+* Change - Provide cache for roles: [#59](https://github.com/owncloud/ocis-pkg/pull/59)
+
+   In order to work efficiently with permissions we provide a cache for roles and a middleware to
+   update the cache based on roleIDs from the metadata context. It can be used to check permissions
+   in service handlers.
+
+   https://github.com/owncloud/ocis-pkg/pull/59
+
+
+* Change - Roles manager: [#60](https://github.com/owncloud/ocis-pkg/pull/60)
+
+   We combined the roles middleware and cache into a roles manager. The manager doesn't expose the
+   cache anymore and manages the state of the cache by fetching roles from the role service which
+   don't exist in the cache, yet.
+
+   https://github.com/owncloud/ocis-pkg/pull/60
+
+# Changelog for [2.4.0] (2020-08-25)
+
+The following sections list the changes in ocis-pkg 2.4.0.
+
+[2.4.0]: https://github.com/owncloud/ocis-pkg/compare/v2.3.0...v2.4.0
+
+## Summary
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (7:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)